### PR TITLE
refac!: Remove sse

### PR DESCRIPTION
**Pull request by gagantrivedi**

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Pull Request
 on:
     push:
         branches:
-            - main #todo - remove
```
Comment (matthewelwell) [inline .github/workflows/pull-request.yml:6]: I think this comment was to remove the workflow from running on pushes to main which does make sense given the name of the workflow file at least.

File: requirements.in [hunk 2]
```diff
@@ -7,7 +7,3 @@ python-dotenv
 pydantic
 orjson
 httpx
-# sse-stuff
-sse-starlette
-asyncio
-redis
```
Comment (matthewelwell) [inline requirements.in:13]: I guess we may want to keep supporting redis as a central cache for the edge proxy... but this is probably something we can handle later down the line.

**Review by matthewelwell — APPROVED** [review]
Approved with 1 minor comment and an observation for future work.

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (gagantrivedi) [reply .github/workflows/pull-request.yml:6]: Hmm, interesting... we do the same thing in core tho

File: requirements.in (hunk 2 above)
Comment (gagantrivedi) [reply requirements.in:13]: Yeah, it's just in memory right now

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (matthewelwell) [reply .github/workflows/pull-request.yml:6]: Yeah, we need to fix that in core as well really but it's fine for now.

**Pull request merged**